CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Welcome to the Fedimint project! We're thrilled that you're interested in contributing to our open-source community. Below are some resources to help you get started:
+
+- **[Contributing Guidelines](https://github.com/fedimint/fedimint/blob/master/docs/contributing.md)**: Read our guidelines to understand how you can contribute to Fedimint.
+
+- **[Development Environment Setup Guide](https://github.com/fedimint/fedimint/blob/master/docs/dev-env.md)**: Follow this guide to set up your development environment.
+
+If you have any questions, don't hesitate to reach out to us on our [Developer Discord](https://discord.gg/BGFMXSkNJW). Our community is friendly and supportive, and we're here to help you succeed.
```

Comment:
Is this a special link that leads directly into a channel? Otherwise I'd propose using `https://chat.fedimint.org`, which is easier to change should we ever need a new invite link.

Reply:
that link  leads directly to `#new-contributors` channel  
while the `https://chat.fedimint.org` leads to general